Subject: [uboot] Enable IP packet defragmentation

tftp is significantly faster if we can use packets larger than
512, e.g. 16KiB or alike.

Signed-off-by: Frank Haverkamp <haver@linux.ibm.com>
---
 include/configs/fsp2.h |    2 ++
 1 file changed, 2 insertions(+)

Index: b/include/configs/fsp2.h
===================================================================
--- a/include/configs/fsp2.h
+++ b/include/configs/fsp2.h
@@ -88,6 +88,8 @@
 #define CONFIG_IPADDR		192.168.0.45
 #define CONFIG_NETMASK		255.255.255.0
 #define CONFIG_SERVERIP		192.168.0.2
+#define CONFIG_IP_DEFRAG	/* IP Defragmentation speeds up tftp */
+
 /*
  * Commands
  */
